sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1740,8 +1740,10 @@ object SQLConf {
 
   val FILES_MIN_PARTITION_NUM = buildConf("spark.sql.files.minPartitionNum")
     .doc("The suggested (not guaranteed) minimum number of split file partitions. " +
-      "If not set, the default value is `spark.default.parallelism`. This configuration is " +
-      "effective only when using file-based sources such as Parquet, JSON and ORC.")
```

Comment:
I think you can just simply replace `spark.default.parallelism` to `spark.sql.leafNodeDefaultParallelism`. If users are interested in, they will read `spark.sql.leafNodeDefaultParallelism`

Reply:
+1

Reply:
Thanks all guys, updated